extensions/azurecore/src/account-provider/auths/networkUtils.ts

```diff
@@ -6,20 +6,6 @@
 import * as azdata from 'azdata';
 import * as https from 'https';
 
-export function getNetworkResponse<Body>(headers: Record<string, string>, body: Body, statusCode: number): NetworkResponse<Body> {
```

Comment:
I didn't really see the need for having a function to just create an object so removed it to reduce the number of hops needed to find where something was defined